res/skins/LateNight/samplers_unit_1234.xml

```diff
@@ -0,0 +1,46 @@
+<Template>
+  <WidgetGroup>
+    <Layout>horizontal</Layout>
+    <SizePolicy>me,max</SizePolicy>
+    <Children>
+
+      <WidgetGroup>
+      <Layout>horizontal</Layout>
+        <Children>
+
+          <Template src="skin:sampler.xml">
+            <SetVariable name="samplernum">1</SetVariable>
+          </Template>
+
+          <WidgetGroup><Size>4f,1min</Size></WidgetGroup>
+
+          <Template src="skin:sampler.xml">
+            <SetVariable name="samplernum">2</SetVariable>
+          </Template>
+
+          <WidgetGroup><Size>4f,1min</Size></WidgetGroup>
+
+          <Template src="skin:sampler.xml">
+            <SetVariable name="samplernum">3</SetVariable>
+          </Template>
+
+          <WidgetGroup><Size>4f,1min</Size></WidgetGroup>
+
+          <Template src="skin:sampler.xml">
+            <SetVariable name="samplernum">4</SetVariable>
+          </Template>
+
+        </Children>
+      </WidgetGroup>
+
+      <WidgetGroup>
+          <Size>5f,1min</Size>
+          <Connection>
+            <ConfigKey>[Master],skin_settings</ConfigKey>
+            <BindProperty>visible</BindProperty>
+          </Connection>
+      </WidgetGroup>
```

Comment:
This spacer is already in samplers_rack.xml, it can be removed here

Reply:
ok, done